mep/people/tests/test_views.py

```diff
@@ -735,6 +735,23 @@ def test_list(self):
 
         # TODO: not sure how to test pagination/multiple pages
 
+        # test partial match
+        response = self.client.get(self.members_url, {'query': "heming"})
+        self.assertContains(response, "Hemingway, Ernest")
+
+        # test accent folding
+        rene_account = Account.objects.create()
+        rene = Person.objects.create(sort_name='Chambrillac, René', slug='rene')
```

Comment:
maybe a dumb question, but does our slugifying logic already do accent folding?

Reply:
Not a dumb question! The migration that added the slugs uses `unidecode` for just that (I think we ran into it before with the derrida project). For new person records we're using the django admin automatic slug generation based on the sort name, which doesn't; I think it just skips characters it can't handle, but that seems ok to me because the person creating the record can correct it.

Reply:
makes sense, thanks for explaining